src/main.rs

```diff
@@ -165,7 +182,18 @@ impl Op {
                     let added_authors = &new_authors - &old_authors;
 
                     if !added_authors.is_empty() {
-                        println!("--> Additional authors ({})", added_authors.iter().join(", "));
+                        println!(
+                            "--> Additional authors ({})",
+                            added_authors.iter().join(", ")
+                        );
+                    }
+
+                    match (get_changelog(old.root()), get_changelog(new.root())) {
+                        (Ok(old), Ok(new)) => {
+                            let diff = changelog_diff(old, new);
+                            println!("--> Additions to CHANGELOG\n{}", diff);
+                        }
+                        _ => (),
```

Comment:
I don't know if I feel comfortable about silently ignoring all errors.

I'd suggest: turn the NotFound errors when opening to empty string (that way we handle crates that just don't have changelogs at all and also the case when a crate starts having a changelog between versions ‒ doing diff against the empty one will yield the full one). This might be best done inside the `get_changelog`.

Then I'd suggest to propagate the rest of the errors properly and bail on them, because anything else except missing changelog is a weird error and the user might want to know.